### PR TITLE
[JSDoc] Add missing documentation for adaptive-expressions/builtinFuntions files 4/10

### PR DESCRIPTION
**Pull request by iscaltritti-sw**
Addresses # 2602

## Description
This PR adds the missing documentation in [adaptive-expressions/builtinFunctions/](https://github.com/microsoft/botbuilder-js/tree/main/libraries/adaptive-expressions/src/builtinFunctions) files.
Some documentation is ported and adapted from [botbuilder-dotnet](https://github.com/microsoft/botbuilder-dotnet)

## Specific Changes
- Added JSDoc comments in all public methods.
- Added `@private` attribute to private methods.
- Fixes linter spacing and semicolon warnings.

## Testing
The following image shows the ESLint Report with no JSDoc errors.
image

### CODE REVIEW

File: libraries/adaptive-expressions/src/builtinFunctions/if.ts [hunk 1]
```diff
@@ -19,10 +19,16 @@ import { ReturnType } from '../returnType';
  * Check whether an expression is true or false. Based on the result, return a specified value.
  */
 export class If extends ExpressionEvaluator {
+    /**
+     * Initializes a new instance of the `GreaterThan` class.
```
Comment (ceciliaavila) [inline libraries/adaptive-expressions/src/builtinFunctions/if.ts:23]:
```suggestion
     * Initializes a new instance of the `If` class.
```

File: libraries/adaptive-expressions/src/builtinFunctions/ignore.ts [hunk 2]
```diff
@@ -20,11 +20,17 @@ import { ReturnType } from '../returnType';
  * 
  */
 export class Ignore extends ExpressionEvaluator {
+    /**
+     * Initializes a new instance of the `GreaterThan` class.
```
Comment (ceciliaavila) [inline libraries/adaptive-expressions/src/builtinFunctions/ignore.ts:24]:
```suggestion
     * Initializes a new instance of the `Ignore` class.
```

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Comment by iscaltritti-sw** [comment]
Fixed, thanks @ceciliaavila !

**Review by sw-joelmut — APPROVED** [review]

**Review by ceciliaavila — APPROVED** [review]

**Review by santgr11 — APPROVED** [review]

**Review by denscollo — APPROVED** [review]

**Comment by denscollo** [comment]
Promoted

**Pull request closed without merging**